docs/docsite/rst/porting_guides/porting_guide_4.rst

```diff
@@ -36,6 +36,8 @@ Other:
 * The configuration system now validates the ``choices`` field, so any settings that currently violate it and are currently ignored will now cause an error.
   For example, `ANSIBLE_COLLECTIONS_ON_ANSIBLE_VERSION_MISMATCH=0` will now cause an error (valid choices are 'ignore', 'warn' or 'error').
 * The ``ansible-galaxy`` command now uses ``resolvelib`` for resolving dependencies. In most cases this should not make a user-facing difference beyond being more performant, but we note it here for posterity and completeness.
+* Python ``module_utils`` imports may now be marked as optional during the module payload build by wrapping the ``import`` statement in a ``try`` or ``if`` block. This allows modules to use ``module_utils`` that may not be present in all versions of Ansible or a collection, and to perform arbitrary recovery or fallback actions during module runtime.
```

Comment:
This is aimed at module developers/maintainers, right? I'd make this a bit clearer, so users can skip this entry with a higher degree of comfort. Maybe something liike:
```suggestion
* If your modules import python ``module_utils``, you may now mark the import as optional during the module payload build by wrapping the ``import`` statement in a ``try`` or ``if`` block. This allows modules to use ``module_utils`` that may not be present in all versions of Ansible or a collection, and to perform arbitrary recovery or fallback actions during module runtime.
```

Reply:
I moved this content to https://github.com/ansible/ansible/pull/74021.